lib/SILOptimizer/Utils/Generics.cpp

```diff
@@ -382,6 +388,12 @@ ReabstractionInfo::createSubstitutedType(SILFunction *OrigF,
     Lowering::GenericContextScope GenericScope(M.Types,
                                                CanSpecializedGenericSig);
     FnTy = OrigF->getLoweredFunctionType()->substGenericArgs(M, SubstMap);
+    // Some of the added new requirements may not have been taken into account
```

Comment:
Can you add a FIXME here so that I can remember to remove this when I check in my fix?